pkg/cloudprovider/fake/cloudprovider.go

```diff
@@ -40,6 +41,10 @@ import (
 	"sigs.k8s.io/karpenter/pkg/utils/resources"
 )
 
+func init() {
+	v1.WellKnownLabels = v1.WellKnownLabels.Insert(v1alpha1.LabelReservationID)
```

Comment:
If you have the reservation ID passed through to identify the offering, I'm thinking you may not need this label anymore to identify the offering

Reply:
You still need it for compatibility checks (e.g. a pod with a node selector on a specific reservation)